blockchain/state_manager/src/vm_circ_supply.rs

```diff
@@ -202,16 +202,14 @@ fn get_fil_power_locked<DB: BlockStore>(
 fn get_fil_reserve_disbursed<DB: BlockStore>(
     state_tree: &StateTree<DB>,
 ) -> Result<TokenAmount, anyhow::Error> {
-    let fil_reserved: BigInt = BigInt::from(300_000_000) * FILECOIN_PRECISION;
+    let fil_reserved: TokenAmount = TokenAmount::from(300_000_000) * FILECOIN_PRECISION;
```

Comment:
Try to keep `TokenAmount` being a consistent unit:
```suggestion
    let fil_reserved: TokenAmount = BigInt::from(300_000_000) * FILECOIN_PRECISION;
```

Reply:
Returned as is was before.